openapi.yml

```diff
@@ -86,12 +86,30 @@ paths:
                 type: object
                 properties:
                   result:
-                    type: object
-                    properties:
-                      trace_id:
-                        type: integer
-                        description: ID of the trace executed
-                        example: 30001
+                    type: array 
+                    items:
+                      type: object
+                      properties:
+                        dpid:
+                          type: string
+                          description: Switch datapath ID 
+                          example: 00:00:00:00:00:00:00:01
+                        port:
+                          type: integer
+                          description: Incoming port in the switch
+                          example: 1
+                        time:
+                          type: string
+                          description: Start time in the first switch
+                          example: "2022-01-25 13:44:52.387021"
+                        type:
+                          type: string
+                          description: Type of the step. May be "starting", "trace", "last" or "loop".
+                          example: "trace"
+                        vlan:
```

Comment:
I've just realized that `vlan` is being added dynamically [in the response depending on a result](https://github.com/kytos-ng/sdntrace_cp/blob/master/main.py#L112-L113), gretel, could you mark this a non-required in this object spec? Just so clients know that `vlan` might or not be in the response

Reply:
I think that, by default, fields are optional unless you put them in the required list. So, in fact all properties are optional, right?

Reply:
Indeed, Gretel. So, let's mark all of them as required and leave `vlan` as it is.

Reply:
Ok.

Reply:
The new commit looks good @gretelliz , let me close this thread. On Swagger UI editor it's also rendering correctly:

![20220907_172020](https://user-images.githubusercontent.com/1010796/188969871-b8558da9-aa10-489f-97d0-37033a247da0.png)

Reply:
Great!